src/sentry/api/endpoints/organization_events.py

```diff
@@ -137,6 +139,28 @@ def get(self, request, organization):
         )
 
 
+class OrganizationEventsTagsEndpoint(OrganizationEventsEndpointBase):
+    def get(self, request, organization, key):
+        try:
+            snuba_args = self.get_snuba_query_args(request, organization)
+        except OrganizationEventsError as exc:
+            return Response({'detail': exc.message}, status=400)
+        except NoProjects:
+            return Response({'detail': 'A valid project must be included.'}, status=400)
+
+        lookup_key = tagstore.prefix_reserved_key(key)
+        project_ids = snuba_args['filter_keys']['project_id']
```

Comment:
Should we also ensure that orgs without `global-views` don't do multi-project queries here too? Like in #13365